lib/msal-angular/docs/v2-v3-upgrade-guide.md

```diff
@@ -0,0 +1,51 @@
+# Upgrading from MSAL Angular v2 to v3
+
+MSAL Angular v3 brings our Angular wrapper up-to-date with the latest version of MSAL common, and with out-of-the-box support for Angular 15 and rxjs 7.
+
+This guide will demonstrate changes needed to migrate an existing application from `@azure/msal-angular` v2 to v3. 
+
+If you are migrating from `@azure/msal-angular` v1, please refer to the [v1-v2 migration guide](./v2-docs/v1-v2-upgrade-guide.md) first to migrate to MSAL v2.
+
+Please also see the [MSAL Browser Migration Doc](../../msal-browser/docs/v2-migration.md) for browser support and other key changes.
+
+## Breaking changes in `@azure/msal-angular@3`
+
+### `MsalRedirectComponent` is now mandatory
+
+MSAL v3.x now requires initializing the application object. Initialization has been built into the `MsalRedirectComponent`, bootstrapping the component is now **MANDATORY**.
+
+See the [guide to redirects](./v2-docs/redirects.md) on how to add `MsalRedirectComponent` to your application.
+
+### The `allowNativeBroker` flag
+
+The `allowNativeBroker` flag is now turned on by default in the configurations. If you're using a B2C authority you can turn it off as follows:
+
+```js
+export function MSALInstanceFactory(): IPublicClientApplication {
+    return new PublicClientApplication({
+        auth: {
+            ...
+        },
+        cache: {
+            ...
+        },
+        system: {
+            allowNativeBroker: false, // Disables native brokering support
+        }
+    });
+}
+```
+
+## Angular 15 and rxjs@7
+
+MSAL Angular now expects that your application is built with `@angular/core@15`, `@angular/common@15`, `rxjs@7`. As with MSAL Angular v2, `rxjs-compat` is not required.
```

Comment:
Should we also note that this isn't backward compatible?

Reply:
Good point. Noted and updated.